diff --git a/CMakeLists.txt b/CMakeLists.txt
index 909d114b9..69229164c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -46,6 +46,10 @@ project(
 )
 set(SNODEC_SOVERSION ${SNode.C_VERSION_MAJOR})
 
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
 set(ignoreMe ${CMAKE_VERBOSE_MAKEFILE})
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index c6e4ee78e..0e2bb6aef 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -79,10 +79,6 @@ if("${SNODEC_INTREE_BUILD}" STREQUAL "YES")
     add_compile_definitions(SNODEC_INTREE_BUILD)
 endif("${SNODEC_INTREE_BUILD}" STREQUAL "YES")
 
-set(CMAKE_CXX_STANDARD 20)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_EXTENSIONS OFF)
-
 add_compile_options(
     -Werror
     -Wall
